vadl/main/vadl/gcb/passes/relocation/model/HasRelocationComputationAndUpdate.java

```diff
@@ -74,4 +74,9 @@ public interface HasRelocationComputationAndUpdate {
    * Generates and returns the name of the ELF relocation.
    */
   ElfRelocationName elfRelocationName();
+
+  /**
+   * Get the LLVM kind of the relocation.
+   */
+  String llvmKind();
```

Comment:
minor: just add a method to return the kind and all the users can call llvmKind themselves.